### DISCUSSION TITLE
On multiple OS types for testing

### DISCUSSION THREAD

**Discussion by kishaningithub** [brianvoe/gofakeit#169] [Q&A] [answered]
I see we are using 3 operating systems for running our tests (list below)

https://github.com/brianvoe/gofakeit/blob/0f4f77fd0a3336303c3fdb634a14306060703b2c/.github/workflows/go.yml#L14

However i don't see OS specific code in this library (correct me if i am wrong)

If feel we can remove mac and windows from the build matrix. What do you think?

**Comment by brianvoe** [comment, marked as answer]
Whether or not there is os specific code being run, I still want to leave it in there in case a specific os has a problem running something
